qml/InitDSMonitorDialog.qml

```diff
@@ -288,46 +292,66 @@ Dialog {
         id: wrongIP
         title: "Invalid Discovery Server IP"
```

Comment:
Add a reference to know if IP is v4 or v6. i.e. `The inserted Discovery Server IP 127.0.0.1 is not a valid IPv6.`